shared/fs/row.js

```diff
@@ -3,44 +3,45 @@ import * as Types from '../constants/types/fs'
 import {connect, type TypedState, type Dispatch} from '../util/container'
 import {isMobile} from '../constants/platform'
 import {navigateAppend} from '../actions/route-tree'
+import type {IconType} from '../common-adapters/icon'
 
 type OwnProps = {
   path: Types.Path,
 }
 
-type StateProps = {
-  type: Types.PathType,
+type DispatchProps = {
```

Comment:
flow can't infer this due to a bug, flow coverage reveals this

Reply:
My bug or a flow bug? What did I miss in flow coverage?

Reply:
Flow bug.